components/Slider/tests/unit/SliderTests.m

```diff
@@ -1166,6 +1195,40 @@ - (void)testNotEnabledHapticFeedback {
   }
 }
 
+- (void)testEnabledFullHapticFeedback {
+  // Given
+  self.slider.minimumValue = 0;
+  self.slider.maximumValue = 5;
+  self.slider.hapticsEnabled = NO;
+  self.slider.numberOfDiscreteValues = 2;
+  self.slider.fullHapticsEnabled = YES;
+
+  if (@available(iOS 10.0, *)) {
+    _mockFeedbackGenerator = [[MockUIImpactFeedbackGenerator alloc] init];
+    self.slider.feedbackGenerator = _mockFeedbackGenerator;
+    for (NSUInteger i = 0; i < 6; ++i) {
+      self.slider.value = i;
+
+      // When
+      [self.slider thumbTrackValueChanged:self.slider.thumbTrack];
+
+      // Then
+      XCTAssertTrue(_mockFeedbackGenerator.impactHasOccurred);
+
+      _mockFeedbackGenerator.impactHasOccurred = NO;
+    }
+  }
+}
+
+- (void)testDefaultHapticsEnabledValues {
```

Comment:
Looks like this is not testing the new property added, is this related more to the previous PR?